cmd/status-cli/main.go

```diff
@@ -0,0 +1,324 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/ethereum/go-ethereum/crypto"
+
+	"go.uber.org/zap"
+
+	"github.com/google/uuid"
+
+	"github.com/status-im/status-go/account/generator"
+	"github.com/status-im/status-go/appdatabase"
+	"github.com/status-im/status-go/common/dbsetup"
+	gethbridge "github.com/status-im/status-go/eth-node/bridge/geth"
+	"github.com/status-im/status-go/eth-node/types"
+	"github.com/status-im/status-go/logutils"
+	"github.com/status-im/status-go/multiaccounts"
+	"github.com/status-im/status-go/protocol"
+	"github.com/status-im/status-go/protocol/common"
+	"github.com/status-im/status-go/protocol/protobuf"
+	"github.com/status-im/status-go/protocol/requests"
+	"github.com/status-im/status-go/t/helpers"
+	"github.com/status-im/status-go/wakuv2"
+	"github.com/status-im/status-go/walletdatabase"
+
+	"github.com/urfave/cli/v2"
+)
+
+const enrBootstrap = "enrtree://AMOJVZX4V6EXP7NTJPMAYJYST2QP6AJXYW76IU6VGJS7UVSNDYZG4@boot.test.shards.nodes.status.im"
+
+var logger *zap.SugaredLogger
+
+type StatusCli struct {
+	name      string
```

Comment:
`name` is only used for logging. Instead we could  add `logger` field here. 
```suggestion
	logger      *zap.Logger
```

And set it as
```go
cli.logger := logger.Named("Alice")
```

Then you will not have to write `[%s]` in each logging line. It will automatically be there.

Reply:
alice and bob already has its own log file for status-go debug log.
`[%s]` is used to better show the processing flow, basically help displaying `who send message to whom` and `who get message from whom`.

Reply:
> [%s] is used to better show the processing flow, basically help displaying who send message to whom and who get message from whom.

Yes. But writing `[%s]` each time is quite annoying and pollutes the code.

What I'm suggesting is to create a separate logger instance for Alice/Bob:
```go
aliceCLI := StatusCLI {
  logger: logger.Named("Alice")
}
bob := StatusCLI {
  logger: logger.Named("Bob")
}
```

then each of `CLI` can use it's own logger. And the naming will already be there.
This is a standard way of naming zap loggers.
```